docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1136,6 +1136,19 @@ For each generated "Trend" field value calculation function can be selected. Def
 
 > **Note:** This transformation is available in Grafana 9.5+ as an opt-in beta feature. Modify Grafana [configuration file][] to use it.
 
+### Regression
+
+Use this transformation to create a new data frame, containing values predicted by a statistical model. This is useful for finding a trend in chaotic data. It works by fitting a mathematical function to the data, using either linear or polynomial regression. The dataframe can then be used in a visualization to display a trendline.
+
+There are two different models:
+
+- **Linear regression** - Fits a linear function to the data.
+  {{< figure src="/static/img/docs/transformations/linear-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+- **Polynomial regression** - Fits a polynomial function to the data.
+  {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+
+**Note:** This mode is an experimental feature. Engineering and on-call support is not available.
```

Comment:
The markdown for this file is now being built from the content in:

- `scripts/docs/generate-transformations.ts` - Includes all content not specific to a transformation. (e.g., link references)
- `public/app/features/transformers/docs/content.ts` - Transformation-specific content.

We did this so that the documentation can now be available in the app itself for referencing. Please update the documentation in these two files, then run the script referred to in the frontmatter of this file to rebuild the markdown in this file. Then both files will be merged when you merge this PR.

I'm still going to make editing suggestions, but these should be implemented in the `content.ts` file.

Also, we have standard language for features that aren't in GA yet, which I've added.

Questions: In the note, should it say "mode" or "model"? Is the word "chaotic" here being used as a mathematical term or just as an adjective? If just as an adjective, can we find a less subjective sounding word?

```suggestion
Use this transformation to create a new data frame containing values predicted by a statistical model. This is useful for finding a trend in chaotic data. It works by fitting a mathematical function to the data, using either linear or polynomial regression. The data frame can then be used in a visualization to display a trendline.

There are two different models:

- **Linear regression** - Fits a linear function to the data.
  {{< figure src="/static/img/docs/transformations/linear-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}
- **Polynomial regression** - Fits a polynomial function to the data.
  {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}

> **Note:** This mode is an experimental feature. Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided. Enable the `regressionTransformation` feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

Reply:
This was generated from `content.ts` but I failed to commit it.

I've updated the docs with your suggestions and made sure to commit `contents.ts` this time :)